source/utilities.tex

```diff
@@ -16056,8 +16054,10 @@
 
 \pnum
 \effects
-Constructs a \tcode{boyer_moore_horspool_searcher} object, initializing \tcode{pat_first_} with \tcode{pat_first},
-\tcode{pat_last_} with \tcode{pat_last}, \tcode{hash_} with \tcode{hf}, and \tcode{pred_} with \tcode{pred}.
+Initializes \tcode{pat_first_} with \tcode{pat_first},
```

Comment:
Should "Initializes" be on its own line? Many changes above do not follow the local convention of not doing so.

Reply:
Fixed.